threedi_api_qgis_client/api_calls/threedi_calls.py

```diff
@@ -87,6 +87,23 @@ def all_simulations_progress(self, simulations_list: List[Simulation]
             progresses[spk] = (sim, current_status, sim_progress)
         return progresses
 
+    def fetch_simulation_results(self, simulation_pk: int) -> List[ResultFile]:
+        """Fetch simulation results list."""
+        api = SimulationsApi(self.api_client)
+        results_list = api.simulations_results_files_list(str(simulation_pk), limit=self.FETCH_LIMIT).results
```

Comment:
does it have some kind of pagination in case of NResults > FETCH_LIMIT?

Reply:
Done.